Formula/gibo.rb

```diff
@@ -1,9 +1,12 @@
+require "date"
+
 class Gibo < Formula
   desc "Access GitHub's .gitignore boilerplates"
   homepage "https://github.com/simonwhitaker/gibo"
-  url "https://github.com/simonwhitaker/gibo/archive/2.2.8.tar.gz"
-  sha256 "07bcc8e7fb4941e095c3740fc4497f0f318cb72c3b0ae83aa13635cefe60ade6"
+  url "https://github.com/simonwhitaker/gibo/archive/v3.0.3.tar.gz"
+  sha256 "d3d76b30d8c61d203304af1805ca3abf77e96c6528f468c4d14830656f982f5c"
   license "Unlicense"
+  version_scheme 1
```

Comment:
no need to have this

Reply:
Thanks. I think I misunderstood that one. I added it because the tagging scheme has changed from x.x.x to vx.x.x (i.e. now has a leading `v`)